runhouse/main.py

```diff
@@ -128,14 +129,47 @@ def ssh(cluster_name: str, up: bool = typer.Option(False, help="Start the cluste
     c.ssh()
 
 
-def _print_status(config):
+###############################
+# Status helping functions
+###############################
+
+
+def _adjust_resource_type(resource_type: str):
+    """
+    status helping function. transforms a str form runhouse.resources.{X.Y...}.resource_type to runhouse.resource_type
+    """
+    if "runhouse" not in resource_type:
```

Comment:
it feels a little brittle to be parsing imports in this way, couldn't we just try importing the `resource_type` (since it should be defined in the main `__init__.py`)?

```
try:
   getattr(importlib.import_module("runhouse"), resource_type)
   return f"runhouse.{resource_type}"
except AttributeError:
 # cannot import the resource type

```